library/ssl_msg.c

```diff
@@ -4011,6 +4011,27 @@ static int ssl_consume_current_message( mbedtls_ssl_context *ssl );
 static int ssl_get_next_record( mbedtls_ssl_context *ssl );
 static int ssl_record_is_in_progress( mbedtls_ssl_context *ssl );
 
+static int ssl_get_record_content( mbedtls_ssl_context *ssl )
```

Comment:
I think this should be an inline function because it is relatively small and called from a loop.

Reply:
@gabor-mezei-arm We have numerous `static` functions in the code which are called from a single place only but not tagged `inline`, relying on the compiler to make the decision to inline it (`inline` is only a recommendation in any case). Is there a reason to believe compilers wouldn't do that in this case? If not, I'd like to not deviate from the existing pattern and keep the code as-is.

Reply:
You are right that for `static` functions the inlining is done implicitly by the compiler and the convention is to omit it where is not necessary.